src/frontend/packages/lib_video/src/api/useTranscriptReaderRequest/index.ts

```diff
@@ -1,5 +1,5 @@
 import { useQuery, UseQueryOptions } from 'react-query';
-import { WebVTT } from 'vtt.js';
+import { VTTCue, WebVTT } from 'vtt.js';
```

Comment:
Is it still needed ?

Reply:
VTTCue not really (but it is used in the file but it probably uses by default de standard one) I don't see any isue to keep it, it was present in the previous implementation (in lti site) https://github.com/openfun/marsha/pull/1884/files#diff-cad9aa0e101f45e8125dd72907fdef8b1880ea66f3deb7cd9754ecf1ace02f97L36

Reply:
ok so we keep it